hash/saturn.xml

```diff
@@ -39035,7 +39035,7 @@ Olympic Soccer (Fra) T-7904H-09
 
 	<!-- Identifying Sonic 3D (Tech Demo - Oct 08)... -->
 	<software name="sonic3dtd" cloneof="sonic3d" supported="no">
-		<description>Sonic 3D (Tech Demo Oct 08)</description>
+		<description>Sonic 3D (tech demo oct 08)</description>
```

Comment:
Month names are proper nouns in English and should be capitalised.

Reply:
Corrected